docs/Antora/modules/getting_started/pages/quick_start_setting_up_csharp.adoc

```diff
@@ -0,0 +1,113 @@
+
+=== Requirements
+
+[tabs]
+====
+Windows::
++
+--
+ * .NET compiler, e.g.
+   - https://learn.microsoft.com/en-us/dotnet/core/sdk[.NET SDK] (used in this guide).
+   - Visual Studio 2022(TM).
+   - Visual Studio Code with extension "C# Dev Kit".
+--
+
+Linux::
++
+--
+ * Ubuntu 22.04 (see also the note below).
+ * On Windows(TM) users can just cross-compile the project for Linux(TM) using the Windows(TM)-requirements (used in this guide).
+ * For development (not tested) and running on Linux(TM): +
+   https://learn.microsoft.com/en-us/dotnet/core/install/linux-ubuntu-install?tabs=dotnet8&pivots=os-linux-ubuntu-2204[Install .NET SDK or .NET Runtime on Ubuntu].
+
+NOTE: On Windows there is the https://learn.microsoft.com/en-us/windows/wsl/install[Windows Subsystem for Linux (WSL)] where the cross-compiled executables can be tested.
+--
+====
+
+In general everything around .NET can be found here: +
+https://dotnet.microsoft.com/en-us/download/dotnet/8.0
+
+=== Creating and building the project
+
+In this guide we use a `Developer PowerShell for VS2022` prompt on Windows(TM) for creating and (cross-) building the project. +
+For the cross-compilation we use a `bash` prompt on Linux(TM) (or WSL on Windows(TM)) to run the result.
+
+To start, create a `csharp/quick_start` directory and navigate to it.
+
+With  the following commands we first create an empty (.NET8.0) console application project and then we add the _openDAQ.Net_ Bindings NuGet package reference to the project, which contains all the needed binaries.
+
+[source,shell]
+----
+dotnet new console
+dotnet add package openDAQ.Net
+----
+
+With this, we are ready to start developing: +
+We fill in our `Program.cs` executable code as follows (replacing the default content):
+
+[source,csharp]
+----
+using Daq.Core.Types;
+using Daq.Core.Objects;
+using Daq.Core.OpenDAQ;
+
+// Create a fresh openDAQ(TM) instance, which acts as the entry point into our application
+Console.WriteLine("create instance...");
+var instance = OpenDAQFactory.Instance();
+
+Console.WriteLine();
+Console.WriteLine($"successfully created the instance '{instance.Name}'");
+
+Console.WriteLine();
+Console.Write("Press a key to exit the application ...");
+Console.ReadKey(intercept: true);
+----
+
+NOTE: When creating the openDAQ(TM) `instance`, we don't need to specify the root directory of our openDAQ(TM) Modules as they are conveniently being deployed by the NuGet package and found where they are.
+
+Now we can compile and run our program in our PowerShell(TM) instance from above.
+
+[tabs]
+====
+Windows::
++
+--
+[source,shell]
+----
+dotnet run
+----
+--
+
+Linux::
++
+--
+Cross-compiling on Windows(TM) and running the result on Linux(TM) using WSL:
+[source,shell]
+----
+dotnet build --runtime linux-x64
+wsl
+cd ./bin/Debug/net8.0/linux-x64/
+./quick_start
```

Comment:
I had to do:
`sudo apt-get install -y dotnet-runtime-8.0`
before I could run `./quick_start` using WSL.
Then everything worked fine.
I'd amend the guide with this info.

Reply:
This is (implicitly) mentioned in the requirements section. I reworked this a bit as well, but it still only refers generally to the installation of the .NET SDK or runtime on Linux/WSL.